lib/google/apis/core/http_command.rb

```diff
@@ -323,6 +334,21 @@ def allow_form_encoding?
 
         private
 
+        def opencensus_begin_span name
+          return unless OPENCENSUS_AVAILABLE && options.use_opencensus
+          return if @opencensus_span
+          return unless OpenCensus::Trace.span_context
+          @opencensus_span = OpenCensus::Trace.start_span name
```

Comment:
Can/should the span be configured with more information, such as the HTTP method?

Reply:
Yes, you're right, there should be a few standard fields added. And I totally forgot, the trace context header should be set as well. Lemme look into that...